docs/Training-Domain-Variation.md

```diff
@@ -1,4 +1,4 @@
-# Training Generalized Reinforcement Learning Agents
+# Domain Variation
```

Comment:
Minor: for non-researcher, the term domain variation may be unintuitive. It may be worth it for us to clarify that a domain can be viewed as an environment (given our terminology around a learning environment).

Reply:
+1
Also title should be "Training with Domain Variation" to match the readme.